Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -167,10 +167,16 @@ def get_discovery_service(self, instance_execute=False) -> dict:
             calling_context = get_calling_context()
             instance_name = calling_context.get('IntegrationInstance', '')
             endpoint = requote_uri(os.path.join('/instance', 'execute', instance_name))
-            service_address = f'{self._url_scheme}://{self._host}{endpoint}'
+            if is_xsiam_or_xsoar_saas():
```

Comment:
Are you sure we are not handling it in other places as well?

Reply:
Not sure but not what I can found.
And without this changed the url is incorrect but with this change it's fine.